content/geometry/general.adoc

```diff
@@ -126,7 +126,7 @@ Users are free to add more keywords for themself, if they are needed. The follow
 * All components shall be named according to capital snake case definition, starting with uppercase letters.
 * Group nodes (also known as empty nodes or parent nodes) shall have "Grp_" as a prefix.
 * Iterators shall be added as suffixes.
-  In the documentation, iterator names are written in angled brackets. Example: <type_idx>.
+  In the documentation, iterator names are written in angled brackets. Example: `<type_idx>``.
```

Comment:
There are two quotation marks at the end.

Reply:
Sorry, that's a typo.